migrate.py

```diff
@@ -178,12 +178,13 @@ def visit_Assign(self, node):
     for fragment in doc_finder.fragments:
         fragment_collection = get_fragment_collection(fragment, spec)
```

Comment:
shouldn't this be 'get_plugin_collection'?

also it is possible there is no collection for the fragment/module_utils so we should not rewrite those